libs/mock-block-dock/dev/test-custom-element-block.ts

```diff
@@ -1,12 +1,44 @@
 import { extractBaseUrl } from "@blockprotocol/graph";
 import { BlockElementBase } from "@blockprotocol/graph/custom-element";
 import { getRoots } from "@blockprotocol/graph/stdlib";
+import { HookBlockHandler } from "@blockprotocol/hook";
 // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Do you know the reason for this import by the way? Can we add a reason to it
```suggestion
// eslint-disable-next-line import/no-extraneous-dependencies -- ?
```

Reply:
The reason for the ignore is that eslint was complaining about `lit` being imported here when it is listed in `devDependencies`, because eslint did not know that this folder is not part of the distributed library. I have updated the linting config in https://github.com/blockprotocol/blockprotocol/pull/1147/commits/499304b6e311d7d8005313ab0dfdd162f766be26 to exclude development files from the rule.

Reply:
Thank you ✨